src/widgets/canvas/line.rs

```diff
@@ -1,5 +1,5 @@
 use crate::{
-    style::Color,
+    prelude::*,
```

Comment:
Seems like a step back? It was more specific and now its less specific? And only seemed to use a single item anyway?

Reply:
1051d9db1